srcpkgs/util-linux/template

```diff
@@ -1,13 +1,14 @@
 # Template file for 'util-linux'
 # Keep this package sync with util-linux-common
 pkgname=util-linux
-version=2.39.3
-revision=2
+version=2.40.1
+revision=1
 build_style=gnu-configure
 configure_args="--exec-prefix=\${prefix} --enable-libuuid --disable-makeinstall-chown
  --enable-libblkid --enable-fsck --disable-rpath --enable-fs-paths-extra=/usr/sbin:/usr/bin
  --enable-vipw --enable-newgrp --enable-chfn-chsh --with-systemdsystemunitdir=no
- --with-udev --without-python --enable-write --disable-libmount-mountfd-support"
+ --with-udev --without-python --enable-write --disable-libmount-mountfd-support
+ --disable-liblastlog2 --disable-pam-lastlog2"
```

Comment:
why disable lastlog2? I see no reason to

Reply:
I read [this](https://karelzak.blogspot.com/) and thought that this is supposed to replace lastlog right away and would rather be done later and decided by the maintainers. Thanks for doing it!
Just for my information, will this not require changes to files in `/etc/pam.d/` for it to start logging in its db?

Reply:
I think to use it it would require a change there, but I don't see any other distro adding it yet